var/spack/repos/builtin/packages/msmpi/package.py

```diff
@@ -69,10 +69,10 @@ def build_command_line(self):
             )
         args.append("/p:IFORT_BIN=%s" % os.path.dirname(ifort_bin))
         args.append("/p:VCToolsVersion=%s" % self.pkg.compiler.msvc_version)
-        args.append("/p:WindowsTargetPlatformVersion=%s" % str(self.pkg.spec["wdk"].version))
-        args.append("/p:PlatformToolset=%s" % self.pkg.compiler.cc_version)
+        args.append("/p:WindowsTargetPlatformVersion=%s" % str(self.pkg.spec["win-wdk"].version))
+        args.append("/p:PlatformToolset=%s" % self.pkg.compiler.cc_version("cl"))
```

Comment:
This lookup should be through the requisite variable in the MSVC class.
If you need the version of `cl.exe`, use `Msvc.cl_version`, for the actual toolset version, use `Msvc.msvc_version`. Otherwise its difficult to know what you mean here.

Reply:
I put the msmpi package file back to its original state. That change is not related to this PR as msmpi must be found externally anyways and this does not fix that issue.